common/aws/provider/connection.go

```diff
@@ -55,6 +55,17 @@ var regionIsValid = func(region string) (isValid bool) {
 	return
 }
 
+var Ticker *time.Ticker
+
+func init() {
```

Comment:
I'm kind of surprised at how infrequently we use this pattern. This seems like a perfect use case.

Reply:
It does, although zpatrick's recent suggestions have made `init()` unnecessary again. xD

I'm still glad that I've learned about it, though.